core/templates/dev/head/pages/exploration_player/ContinueButtonDirective.js

```diff
@@ -0,0 +1,33 @@
+  // Copyright 2017 The Oppia Authors. All Rights Reserved.
```

Comment:
Why are these copyright lines indented by 2? They should not be indented at all. (See other files.)